app/data/DadosNotification.ts

```diff
@@ -1,6 +1,40 @@
-export const dadosNotification: [title: string, about: string, date: string][] = [
-  ["Aumento", "CDB", "12/09"],
-  ["Redução", "Bitcoin", "28/08"],
-  ["Aumento", "Ações", "11/08"],
-  ["Redução", "CDI", "16/07"],
+export const dadosNotification: [
+  date: Date,
+  icon: string,
+  title: string,
+  about: string,
+  details: string,
+][] = [
+  [new Date(2025, 8, 29, 17, 0, 0, 0), "bell", "Lembrete!", "Lorem ipsum dolor sit amet. ", ""],
+  [
+    new Date(2025, 8, 29, 17, 0, 0, 0),
+    "bell",
+    "Nova Atualização",
+    "Lorem ipsum dolor sit amet. ",
+    "",
+  ],
+  [
+    new Date(2025, 8, 29, 17, 0, 0, 0),
+    "bell",
+    "Transações",
+    "Lorem ipsum dolor sit amet. ",
+    "Mercado |  padaria  |  -$100,00",
+  ],
+  [new Date(2025, 8, 28, 17, 0, 0, 0), "bell", "Lembrete!", "Lorem ipsum dolor sit amet. ", ""],
+  [new Date(2025, 8, 24, 17, 0, 0, 0), "bell", "Despesas", "Lorem ipsum dolor sit amet. ", ""],
+]
+
+export const mesesAno: [mes: string][] = [
```

Comment:
```suggestion
export const mesesAno: string[] = [
```
Pra que usar ["Janeiro"] ao invés de "Janeiro"? kkk